basics.md

```diff
@@ -65,18 +65,19 @@ concepts behind {program}`tskit`, the tree sequence toolkit.
 ## Terminology
 
 ::::{margin}
-:::{seealso}
-Many of the terms below are also {ref}`covered <tskit:sec_data_model_definitions>`
-in the official {program}`tskit` docs
+:::{note}
+See {ref}`here <sec_intro_downloading_datafiles>` to run the code in this tutorial on
```

Comment:
Longer but possibly better: "to get the files that will allow you to run the code in this tutorial on your own computer"